spec/Parse.Push.spec.js

```diff
@@ -127,6 +127,7 @@ describe('Parse.Push', () => {
           alert: 'Hello world!'
         }
       }, {useMasterKey: true}))
+      .then(() => delayPromise(500))
```

Comment:
can these be any shorter?

Reply:
nope, not really, because we don't wait till it's completed...